src/main/java/com/angelbroking/smartapi/utils/Constants.java

```diff
@@ -1,43 +1,204 @@
 package com.angelbroking.smartapi.utils;
 
+
 /**
  * Contains all the Strings that are being used in the Smart API Connect library.
  */
 public class Constants {
 
-    /** Product types. */
-	public static String PRODUCT_DELIVERY = "DELIVERY";
-	public static String PRODUCT_INTRADAY = "INTRADAY";
-	public static String PRODUCT_MARGIN = "MARGIN";
-	public static String PRODUCT_BO = "BO";
-	public static String PRODUCT_CARRYFORWARD = "CARRYFORWARD";
-
-    /** Order types. */
-    public static String ORDER_TYPE_MARKET = "MARKET";
-    public static String ORDER_TYPE_LIMIT = "LIMIT";
-    public static String ORDER_TYPE_STOPLOSS_LIMIT = "STOPLOSS_LIMIT";
-    public static String ORDER_TYPE_STOPLOSS_MARKET = "STOPLOSS_MARKET";
-
-    /** Variety types. */
-    public static String VARIETY_NORMAL = "NORMAL";
-    public static String VARIETY_AMO = "AMO";
-    public static String VARIETY_STOPLOSS = "STOPLOSS";
-    public static String VARIETY_ROBO = "ROBO";
-    
-    /** Transaction types. */
-    public static String TRANSACTION_TYPE_BUY = "BUY";
-    public static String TRANSACTION_TYPE_SELL = "SELL";
-
-    /** Duration types. */
-    public static String DURATION_DAY = "DAY";
-    public static String DURATION_IOC = "IOC";
-
-    /** Exchanges. */
-    public static String EXCHANGE_NSE = "NSE";
-    public static String EXCHANGE_BSE = "BSE";
-    public static String EXCHANGE_NFO = "NFO";
-    public static String EXCHANGE_CDS = "CDS";
-    public static String EXCHANGE_NCDEX = "NCDEX";
-    public static String EXCHANGE_MCX = "MCX";
+    // Private constructor to prevent instantiation from outside the class
+    private Constants() {
+        // This constructor is intentionally left blank
+    }
+
+    /**
+     * Product types.
+     */
+    public static final String PRODUCT_INTRADAY = "INTRADAY";
+
+
+    /**
+     * Order types.
+     */
+    public static final String ORDER_TYPE_STOPLOSS_LIMIT = "STOPLOSS_LIMIT";
+
+    /**
+     * Variety types.
+     */
+    public static final String VARIETY_STOPLOSS = "STOPLOSS";
+
+    /**
+     * Transaction types.
+     */
+    public static final String TRANSACTION_TYPE_BUY = "BUY";
+
+    /**
+     * Duration types.
+     */
+    public static final String DURATION_DAY = "DAY";
+
+    /**
+     * Exchanges.
+     */
+    public static final String EXCHANGE_NSE = "NSE";
+
+
+    /**
+     * SmartStream Routes
+     */
+    public static final String ROOT_URL = "https://apiconnect.angelbroking.com";
+    public static final String LOGIN_URL = "https://apiconnect.angelbroking.com/rest/auth/angelbroking/user/v1/loginByPassword";
+    public static final String WSURI = "wss://wsfeeds.angelbroking.com/NestHtml5Mobile/socket/stream";
+//    public static final String SMARTSTREAM_WSURI = "ws://smartapisocket.angelone.in/smart-stream";
+    public static final String SMARTSTREAM_WSURI = "ws://mds-uat.angelone.in/smart-stream";

```

Comment:
this is UAT URL. Please switch to prod URL

Reply:
sure